packages/video_player/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: video_player_tizen
+description: Flutter plugin for displaying inline video with other Flutter
+  widgets on Tizen.
+version: 2.0.0
```

Comment:
So this could be 1.0.0. It's up to your decision.

Reply:
Ther version has benn changed to 1.0.0